plugins/time/pluma-time-plugin.c

```diff
@@ -177,227 +177,227 @@ static void time_cb (GtkAction *action, PlumaTimePlugin *plugin);
 }
 
 static void
 pluma_time_plugin_deactivate (PeasActivatable *activatable)
 {
-	PlumaTimePluginPrivate *data;
-	PlumaWindow *window;
-	GtkUIManager *manager;
+        PlumaTimePluginPrivate *data;
+        PlumaWindow *window;
+        GtkUIManager *manager;
 
-	pluma_debug (DEBUG_PLUGINS);
+        pluma_debug (DEBUG_PLUGINS);
 
-	data = PLUMA_TIME_PLUGIN (activatable)->priv;
-	window = PLUMA_WINDOW (data->window);
+        data = PLUMA_TIME_PLUGIN (activatable)->priv;
+        window = PLUMA_WINDOW (data->window);
 
-	manager = pluma_window_get_ui_manager (window);
+        manager = pluma_window_get_ui_manager (window);
 
-	gtk_ui_manager_remove_ui (manager, data->ui_id);
-	gtk_ui_manager_remove_action_group (manager, data->action_group);
+        gtk_ui_manager_remove_ui (manager, data->ui_id);
+        gtk_ui_manager_remove_action_group (manager, data->action_group);
 }
 
 static void
 pluma_time_plugin_update_state (PeasActivatable *activatable)
 {
-	pluma_debug (DEBUG_PLUGINS);
+        pluma_debug (DEBUG_PLUGINS);
 
-	update_ui (PLUMA_TIME_PLUGIN (activatable)->priv);
+        update_ui (PLUMA_TIME_PLUGIN (activatable)->priv);
 }
 
 /* whether we should prompt the user or use the specified format */
 static PlumaTimePluginPromptType
 get_prompt_type (PlumaTimePlugin *plugin)
 {
-	PlumaTimePluginPromptType prompt_type;
+        PlumaTimePluginPromptType prompt_type;
 
-	prompt_type = g_settings_get_enum (plugin->priv->settings,
-			        	       PROMPT_TYPE_KEY);
+        prompt_type = g_settings_get_enum (plugin->priv->settings,
+                                               PROMPT_TYPE_KEY);
 
-	return prompt_type;
+        return prompt_type;
 }
 
 static void
 set_prompt_type (GSettings *settings,
-		 PlumaTimePluginPromptType  prompt_type)
+                 PlumaTimePluginPromptType  prompt_type)
 {
-	if (!g_settings_is_writable (settings,
-					   PROMPT_TYPE_KEY))
-	{
-		return;
-	}
-
-	g_settings_set_enum (settings,
-				 PROMPT_TYPE_KEY,
-				 prompt_type);
+        if (!g_settings_is_writable (settings,
+                                           PROMPT_TYPE_KEY))
+        {
+                return;
+        }
+
+        g_settings_set_enum (settings,
+                                 PROMPT_TYPE_KEY,
+                                 prompt_type);
 }
 
 /* The selected format in the list */
 static gchar *
 get_selected_format (PlumaTimePlugin *plugin)
 {
-	gchar *sel_format;
+        gchar *sel_format;
 
-	sel_format = g_settings_get_string (plugin->priv->settings,
-					      SELECTED_FORMAT_KEY);
+        sel_format = g_settings_get_string (plugin->priv->settings,
+                                              SELECTED_FORMAT_KEY);
 
-	return sel_format ? sel_format : g_strdup (formats [0]);
+        return sel_format ? sel_format : g_strdup (formats [0]);
 }
 
 static void
 set_selected_format (GSettings *settings,
-		     const gchar     *format)
+                     const gchar     *format)
```

Comment:
Indents..........or delete this commit ;)

Ok, i see there are more issues.
This commit doesn't make any sense for me....
I am still surprise that someone could approve that........